ssn/integrated/informativeExamples/InkBird-IBS-TH2-instance.ttl

```diff
@@ -0,0 +1,267 @@
+    sosa:hasFeatureOfInterest <acmePorterSixPackAirSample/10/PO202402/21/0001> ;
+    sosa:resultTime "2024-02-20T01:35:45Z"^^xsd:dateTime ;
+    sosa:hasSimpleResult "60"^^unit:PERCENT .
+
+# The battery is a special case because it's property is linked to the battery, not to the environment.
+
+<observation/1c> rdf:type sosa:Observation ;
+    rel:own <acmeBreweryCo/1> ;
+    sosa:observedProperty <BatteryHealth/12345> ;
+    sosa:madeBySensor <BatterySensor/12345>  ;
+    sosa:hasFeatureOfInterest <batteryPack/1> ;
+    sosa:resultTime "2024-02-20T01:35:45Z"^^xsd:dateTime ;
+    sosa:hasSimpleResult "99"^^unit:PERCENT .
+
+<breweryObserver/1> a prov:Activity ;
+    rdfs:comment "Brewery operating system logging process." ;
+    prov:atLocation <acmeBreweryCo/1> ;
+    prov:generated <observation/1a> ;
+    prov:generated <observation/1b> ;
+    prov:generated <observation/1c> ;
+    prov:wasStartedBy <alice/1> . # She turned it on last time.     
+
+#
+# Product is loaded on the truck
+#
+
+    
+<observation/67a> a sosa:Observation, prov:Entity ;
+    rel:own <acmeBreweryCo/1> ;
+    rel:own <acmeDelivery/1> ;
+    sosa:observedProperty <BeerTemperature/10/PO202402/21/0001> ;
+    sosa:hasUltimateFeatureOfInterest <acmePorterSixPackBeerSample/10/PO202402/21/0001> ;
+    sosa:madeBySensor <TemperatureSensor/12345>  ;
+    sosa:hasFeatureOfInterest <acmePorterSixPackAirSample/10/PO202402/21/0001> ;
+    sosa:resultTime "2024-02-22T04:15:05Z"^^xsd:dateTime ;
+    sosa:hasSimpleResult "19"^^unit:DEG_C .
+
+<observation/67b> rdf:type sosa:Observation,prov:Entity ;
+    rel:own <acmeBreweryCo/1> ;
+    rel:own <acmeDelivery/1> ;
+    sosa:observedProperty equipment:airRelativeHumidity ;
+    sosa:hasUltimateFeatureOfInterest <ProductPackaging/1>;
+    sosa:madeBySensor <HumiditySensor/12345>  ;
+    sosa:hasFeatureOfInterest <acmePorterSixPackAirSample/10/PO202402/21/0001> ;
+    sosa:resultTime "2024-02-22T04:15:05Z"^^xsd:dateTime ;
+    sosa:hasSimpleResult "74"^^unit:PERCENT .
+
+<observation/67c> rdf:type sosa:Observation,prov:Entity ;
+    rel:own <acmeBreweryCo/1> ;
+    rel:own <acmeDelivery/1> ;
+    sosa:observedProperty <BatteryHealth/12345>;
+    sosa:madeBySensor <BatterySensor/12345>  ;
+    sosa:hasFeatureOfInterest <batteryPack/1> ;
+    sosa:resultTime "2024-02-22T04:15:05Z"^^xsd:dateTime ;
+    sosa:hasSimpleResult "99"^^unit:PERCENT .
+
+<TruckObserver/1> a prov:Activity ;
+    rdfs:comment "Truck onboard monitoring system" ;
+    prov:used <acmeTruck/1> ;
+    prov:generated <observation/67a> ;
+    prov:generated <observation/67b> ;
+    prov:generated <observation/67c> ;
+    prov:startedAtTime "2024-02-22T04:15:05Z"^^xsd:dateTime ;
+    prov:endedAtTime "2024-02-22T05:55:38Z"^^xsd:dateTime ;
+    prov:wasStartedBy <driver/1> ;
+    prov:atLocation <gpsLocation/1> .
+
+# This is the output from the truck's GPS receiver. We only get coordinates
+# from it.
+
+<gpsLocation/1> a prov:Location, geosparql:Geometry; 
```

Comment:
Neater if it is an instance of geosparql:Point instead.

Reply:
There is no geosparql:Point, you're thinking of a GML Point... There is a geosparql:Centroid which isn't quite what a GPS reports.

Reply:
Sorry, was referring to `sf:Point` from the [Sampling Features](https://opengeospatial.github.io/ogc-geosparql/geosparql11/sf_geometries.ttl) ontology. It as a sub-class of `geo:Geometry`.

Reply:
It is done.